OpenDreamRuntime/Objects/DreamObjectTree.cs

```diff
@@ -364,6 +364,18 @@ public sealed class DreamObjectTree {
                     objectDefinition.GlobalVariables.Add(jsonGlobalVariable.Key, jsonGlobalVariable.Value);
                 }
             }
+
+            if (jsonObject.ConstVariables != null) {
+                foreach (string jsonConstVariable in jsonObject.ConstVariables) {
+                    objectDefinition.ConstVariables.Add(jsonConstVariable);
+                }
```

Comment:
You can make `DreamObjectDefinition.ConstVariables` not readonly and just do `objectDefinition.ConstVariables = jsonObject.ConstVariables` instead of this loop. Maybe also make it nullable and start the value off as null since a lot of objects aren't going to have const vars.